lib/analyzer/package-manager/apk.ts

```diff
@@ -1,19 +1,14 @@
-import { Docker, DockerOptions } from "../docker";
-import { AnalyzerPkg } from "./types";
-export { analyze };
+import { AnalyzerPkg, AnalyzerResult } from "../types";
```

Comment:
Perhaps we could use plural folder form `lib/analyzer/package-managers`, like we used for `lib/inputs`?

Reply:
Done!